bundle-size/__mocks__/db.js

```diff
@@ -1,3 +1,18 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
All these files are so important they deserve TWO copyrights! 😅
(also, don't change the year)

Reply:
Yeah this is what `eslint fix` does. I just revived this branch/PR but can't revert it to a draft AFAICT. Needs some cleanup :)